packages/prisma/migrations/20220328185001_soft_delete_booking_references/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "BookingReference" ADD COLUMN     "deleted" BOOLEAN;
```

Comment:
Same here with `deletedAt`

Reply:
The thing was setting up deletedAt: Date with prisma was kinda hard, and preferred boolean for simplicity.